packages/grafana-e2e/src/flows/configurePanel.ts

```diff
@@ -114,11 +114,9 @@ export const configurePanel = (config: PartialAddPanelConfig | PartialEditPanelC
       }
     }
 
-    e2e().server();
-
     // @todo alias '/**/*.js*' as '@pluginModule' when possible: https://github.com/cypress-io/cypress/issues/1296
 
-    e2e().route(chartData.method, chartData.route).as('chartData');
+    e2e().intercept(chartData.method, chartData.route).as('chartData');
```

Comment:
Those this cause a lot of trouble for you @vickyyyyyyy? I can't test this.

Reply:
I don't think so - I ran our enterprise plugins e2e tests with this and it looks fine 👍